lib/generators/sorcery/templates/initializer.rb

```diff
@@ -250,6 +250,12 @@
     #
     # user.subclasses_inherit_config =
 
+    # Set token randomness.
+    # The length of the result string is about 4/3 of `token_randomness`.
+    # Default: `15`
+    #
+    # config.token_randomness =
```

Comment:
@kyuden 

Fixed 🙇 

-------------

I confirmed that the token in password reset email will become longer:

<img width="939" alt="screen shot 2017-06-18 at 20 37 45" src="https://user-images.githubusercontent.com/1796864/27260390-10e91ea8-5466-11e7-9b08-4188d0dde2a2.png">

<img width="335" alt="screen shot 2017-06-18 at 20 38 28" src="https://user-images.githubusercontent.com/1796864/27260394-1e77c704-5466-11e7-88e7-d275b94f091f.png">